src/main/java/com/example/task7/SampleController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.task7;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Map;
+
+@RestController
+public class SampleController {
+
+    @GetMapping("/names")
+    public CreateForm createForm(@RequestParam(value = "name", defaultValue = "name") String name, @RequestParam(value = "birthday", defaultValue = "0000") String birthday) {
+        return new CreateForm(String.format(name), String.format(birthday));
+    }
```

Comment:
名前検索を意図していますか？
DBが絡んでくるところで本格対応でも大丈夫です。
ユーザーを扱うAPIと仮定します。
```suggestion
    @GetMapping("/names")
    public UserResponse search(@RequestParam(value = "name", defaultValue = "name") String name, @RequestParam(value = "birthday", required = false) String birthday) {
        return new CreateForm(String.format(name), String.format(birthday));
    }
```
リクエストされたnameとbirthdayで検索をするようなAPIを作成するといいかなと思います。
繰り返しになりますが、DBの学習後着手でいいと思います。